### DISCUSSION TITLE
SAIC URI Database

### DISCUSSION THREAD

**Discussion by Tonno87** [SAIC-iSmart-API/documentation#8] [General]
Hi, i think we should collect some working SAIC_URI's in this thread:

1.  Europe: https://tap-eu.soimt.com (E-Mail or Phone)
2.  Australia: https://tap-au.soimt.com
3.  India: https://iov-tap.mgindia.co.in (Phone only) <-- not tested yet
4.  Thailand: https://iov-tap.mgthai.com (Phone only with 50 characters (e.g. 00000000000000000000000000000000000000000123456789)

**Comment by tisoft** [comment]
Thailand: https://iov-tap.mgthai.com

**Reply by tisoft** [reply]
Thailand only supports phone numbers. The format is a bit different to EU/AU/NZ. With the Java MQTT Gateway it should work to prefix your phone number with `0`'s until the whole string is exactly 50 characters long. (e.g.`00000000000000000000000000000000000000000123456789`) I haven't checked if the Python MQTT Gateway would accept that. 

I assume the same format should be working in the India version, but all versions of the India App I could find, are encrypted and I can't get the source code, so I can't verify.

**Reply by tisoft** [reply]
Found an older unencrypted India app. The login code is the same as in the Thai App.

**Comment by nickh66** [comment]
I'm in Australia & i'm trying the AU URI listed here.  I can ping this so it seems valid but logon with my SAIC credentials i used in the MG iSmart app seem to not work.  Does anyone know how to confirm functionality of this URI?
